File: SW Expert Academy/D2/01945. 간단한 소인수분해/간단한 소인수분해.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
	int T; cin>>T;
	for(int tc=0; tc<T; tc++){
		int N; cin>>N;
		vector<int> divisor = {2, 3, 5, 7, 11};
		
		printf("#%d ", tc+1);
		for(int i=0; i<divisor.size(); i++){
			int cnt = 0;
			while(!(N%divisor[i])){
				cnt++;
				N /= divisor[i];
			}
			printf("%d ", cnt);
		}
		cout<<endl;
	}
}

File: SW Expert Academy/D2/01945. 간단한 소인수분해/간단한 소인수분해.py
for tc in range(int(input())):
    N = int(input())
    divisor = [2, 3, 5, 7, 11]

    print("#%d"%(tc+1), end=' ')
    for div in divisor:
        cnt = 0
        while N%div == 0:
            cnt += 1
            N //= div

        print(cnt, end=' ')
    print()
